narwhals/_arrow/series.py

```diff
@@ -185,7 +185,7 @@ def __rtruediv__(self, other: Any) -> Self:
         pc = get_pyarrow_compute()
         ser = self._native_series
         other = validate_column_comparand(other)
-        if not isinstance(other, (pa.Array, pa.ChunkedArray)):
+        if not isinstance(other, (pa.Array, pa.ChunkedArray)):  # pragma: no cover
```

Comment:
thanks for doing this!

just a question about this line, why is it no longer covered?